spec/factories/packages.rb

```diff
@@ -7,6 +7,7 @@
     width       { rand(199) + 1 }
     height      { rand(199) + 1 }
     notes       { Faker::Lorem.paragraph }
+    state       'expecting'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.